File: src/com/l2kt/gameserver/network/serverpackets/PartyMatchDetail.java
package com.l2kt.gameserver.network.serverpackets;

import com.l2kt.gameserver.model.partymatching.PartyMatchRoom;

/**
 * @author Gnacik
 */
public class PartyMatchDetail extends L2GameServerPacket
{
	private final PartyMatchRoom _room;
	
	public PartyMatchDetail(PartyMatchRoom room)
	{
		_room = room;
	}
	
	@Override
	protected final void writeImpl()
	{
		writeC(0x97);
		writeD(_room.getId()); // Room ID
		writeD(_room.getMaxMembers()); // Max Members
		writeD(_room.getMinLvl()); // Level Min
		writeD(_room.getMaxLvl()); // Level Max
		writeD(_room.getLootType()); // Loot Type
		writeD(_room.getLocation()); // Room Location
		writeS(_room.getTitle()); // Room title
	}
}

File: src/com/l2kt/gameserver/network/serverpackets/PartyMatchDetail.kt
package com.l2kt.gameserver.network.serverpackets

import com.l2kt.gameserver.model.partymatching.PartyMatchRoom

/**
 * @author Gnacik
 */
class PartyMatchDetail(private val _room: PartyMatchRoom) : L2GameServerPacket() {

    override fun writeImpl() {
        writeC(0x97)
        writeD(_room.id) // Room ID
        writeD(_room.maxMembers) // Max Members
        writeD(_room.minLvl) // Level Min
        writeD(_room.maxLvl) // Level Max
        writeD(_room.lootType) // Loot Type
        writeD(_room.location) // Room Location
        writeS(_room.title) // Room title
    }
}